install.ps1

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env pwsh
+
+$ErrorActionPreference = 'Stop'
+
+$JsctlInstall = $env:JSCTL_INSTALL
+$BinDir = if ($JsctlInstall) {
+  "${JsctlInstall}\bin"
+} else {
+  "${Home}\.jsctl\bin"
+}
+
+$JsctlTar = "$BinDir\jsctl.tar.gz"
+$JsctlExe = "$BinDir\jsctl.exe"
+
+$Target = if ($ENV:OS -eq "Windows_NT") {
+    $arch = if ($ENV:PROCESSOR_ARCHITEW6432) {
+        $ENV:PROCESSOR_ARCHITEW6432
+    } else {
+        $ENV:PROCESSOR_ARCHITECTURE
+    }
+
+    switch ($arch) {
+        "AMD64" { "jsctl-amd64-windows" }
+        "ARM64" { "jsctl-arm64-windows" }
+        default { throw "Error: Unsupported windows achitecture: ${arch}" }
+    }
+} else {
+    throw "Error: Unsupported operating system, use the install.sh script instead."
+}
+
+$DownloadUrl = "https://github.com/jetstack/jsctl/releases/download/latest/${Target}.tar.gz"
```

Comment:
I think this might need to be `https://github.com/jetstack/jsctl/releases/latest/download/${Target}.tar.gz` instead
(see i.e that https://github.com/jetstack/jsctl/releases/latest/download/jsctl-0.1.15-linux-x86_64.tar.gz)